cmd/version/version.go

```diff
@@ -83,7 +83,7 @@ func (o *option) SetWriter(writer io.Writer) {
 func NewCMD() *cobra.Command {
 	o := &option{
 		outputFormat: cli.NewOutputFormat().
-			WithDefaultFormat(printer.NewJSON().Format()).
+			WithDefaultFormat(printer.NewYAML().Format()).
```

Comment:
why is this changing the default to yaml?

Reply:
At the time of writing the PR originally, the test was checking for YAML output since that's what was being used ([commit](https://github.com/aurae-runtime/ae/commit/cd721228387bb8ddc3e2889ddf63b0a0e1bef75b#diff-a6e9cedf36ae1f9b807d361a421ec6a917637b9e11630fde62ded50f8b547d85R46-R52) ), then the the OutputFormat PR changed it to JSON which broke the test. This change was just to keep the tests the same and yaml did seem more natural for output from a `version` command. I can change it back to json if you want, I'm good with either.

Reply:
yes please. our README suggests we should be moving towards JSON as output across the board. earlier flexibility to allow yaml was useful but not something we want to maintain going forward.

Reply:
Updated 👍